applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -349,23 +309,22 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
 
                     // check this if statement, I don't understand the check for pipe erosion
                     if (((!OpenPipeElement->GetValue(PIPE_EROSION) || (current_height > eq_height)) &&
-                         current_height < amax)) {
+                         current_height < MaxPipeHeight)) {
                         OpenPipeElement->SetValue(PIPE_HEIGHT, OpenPipeElement->GetValue(PIPE_HEIGHT) + da);
                         equilibrium = false;
                     }
 
                     // check if equilibrium height and current pipe heights are diverging, stop
-                    // picard iterations if this is the case and set pipe height on zero
-                    if (!OpenPipeElement->GetValue(PIPE_EROSION) && (PipeIter > 1) &&
+                    // Picard iterations if this is the case and set pipe height on zero
+                    if (!OpenPipeElement->GetValue(PIPE_EROSION) && (piping_iteration > 1) &&
                         ((eq_height - current_height) > OpenPipeElement->GetValue(DIFF_PIPE_HEIGHT))) {
                         equilibrium = true;
                         OpenPipeElement->SetValue(PIPE_HEIGHT, small_pipe_height);
                     }
                     // calculate difference between equilibrium height and current pipe height
                     OpenPipeElement->SetValue(DIFF_PIPE_HEIGHT, eq_height - current_height);
                 }
-                // increment piping iteration number
-                PipeIter += 1;
+                piping_iteration += 1;
```

Comment:
The more natural way to increment in C++ is to use `++`:
```suggestion
                ++piping_iteration;
```

Reply:
of course.